src/screens/AddNewTerm.js

```diff
@@ -21,15 +21,15 @@ export default ({navigation, categories, addNewPhrase}) => {
   const [addEnglishPhrase, setAddEnglishPhrase] = useState('');
   const [addMalagasyPhrase, setAddMalagasyPhrase] = useState('');
 
-  const isButtonEnable = addEnglishPhrase === '' || addMalagasyPhrase === '' || selectedCategory === '';
+  const isButtonEnable =
```

Comment:
This does not seem to be part of the learnt phrases user story and therefore should not be part of the PR